README.md

```diff
@@ -1,3 +1,99 @@
-# Node.js Express Web Service using standard `node` image
+# Hardening Docker Workloads using Slim CLI
```

Comment:
It's kinda handy to have a sentence per line in markdown files. Helps with reviews and change tracking.

Reply:
Done